tests/regression/requirements.txt

```diff
@@ -1 +1 @@
-ftw==1.1.7
+ftw==1.2.2
```

Comment:
This will work but we might want to use 1.2.3

Reply:
Updated based on pypi release.